code/game/objects/structures/window.dm

```diff
@@ -309,10 +318,29 @@ var/list/one_way_windows
 			one_way_windows = list()
 		one_way_windows.Add(src)
 		P.use(1)
-		to_chat(user, "<span class='notice'>You place a sheet of plastic over the window.</span>")
+		to_chat(user, "<span class='notice'>You place a sheet of plastic over the [src].</span>")
```

Comment:
\the [src] or you're gonna get the the window

Reply:
I still have a hard time understanding how they could have possibly thought that was a good idea